lib/buffer.js

```diff
@@ -30,18 +30,22 @@ function alignPool() {
 }
 
 
-function Buffer(arg) {
+function Buffer(arg, zeroFill) {
```

Comment:
don't like having an argument named for one specific usage that is used otherwise. check `arguments[1]`, unless other's are alright w/ something else generic.